### ISSUE TITLE
0.53.0 is not available on Cocoapods

### ISSUE THREAD

**Issue by DmitrySerovWise** [realm/SwiftLint#5240]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

The version 0.53.0 was released yesterday, but is still nowhere to be found in the Cocoapods official spec, meaning that `pod update` doesn't see the new version.

Previous versions were available the same day, so I guess something went wrong.

##### Complete output when running SwiftLint, including the stack trace and command used

Can't install to check as the newest version is not available.

### Environment

* SwiftLint version 0.52.4
* Installation method used: Cocoapods

**Comment by jpsim** [comment]
Yes something went wrong. Cocoapods threw an error related to xrOS (I had Xcode 14.3.1 selected via `xcode-select` though) and I gave up after a few minutes of trying to fix it yesterday. I’ll look into it some more today.
